src/modules/Elsa.Workflows.Core/Services/WorkflowRunner.cs

```diff
@@ -66,15 +67,19 @@ public async Task<RunWorkflowResult<TResult>> RunAsync<TResult>(WorkflowBase<TRe
     }
 
     /// <inheritdoc />
-    public async Task<RunWorkflowResult> RunAsync<T>(RunWorkflowOptions? options = default, CancellationToken cancellationToken = default) where T : IWorkflow
+    public async Task<RunWorkflowResult> RunAsync<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] T>(
```

Comment:
We need this attribute due to an underlying function, which means if we want to be consistent we need to propogate this all the way up, cluttering a lot of our code with these attributes.
Instead of this attribute we can also put the new() constraint on T

Reply:
Really - I didn't know. I will try.

Reply:
You are right, that worked! Let's remove the attribute whenever we come across it and fix it with the new() constraint.